core/lib/zksync_core/src/api_server/web3/namespaces/debug.rs

```diff
@@ -79,7 +79,8 @@ impl DebugNamespace {
         let call_trace = connection
             .blocks_web3_dal()
             .get_trace_for_miniblock(block_number)
-            .await;
+            .await
+            .unwrap();
```

Comment:
The error can be mapped here. Will this be done in a separate PR?

Reply:
I'm doing error propagation by layer-by-layer. Currently I'm just pushing out the panics out of the dal crate.